docs/guides/managed-backup/dedicated-backup-namespace-auto-backup/index.md

```diff
@@ -0,0 +1,407 @@
+---
+title: Define Cluster Wide Backup Policy using Stash Auto-backup
+description: A guide on how to define cluster wide backup policy using Stash auto-backup.
+menu:
+  docs_{{ .version }}:
+    identifier: managed-backup-dedicated-backup-namespace-auto-backup
+    name: Define Cluster Wide Backup Policy using Stash Auto-backup
+    parent: managed-backup
+    weight: 30
+product_name: stash
+menu_name: docs_{{ .version }}
+section_menu_id: guides
+---
+
+# Define Cluster Wide Backup Policy using Stash Auto-backup
+
+This guide will show you how you can define cluster wide backup policy using Stash Auto-backup.
+
+## Before You Begin
+
+- At first, you need to have a Kubernetes cluster, and the `kubectl` command-line tool must be configured to communicate with your cluster. If you do not already have a cluster, you can create one by using [kind](https://kind.sigs.k8s.io/docs/user/quick-start/).
+
+- Install `Stash` in your cluster following the steps [here](/docs/setup/README.md).
+
+- You should be familiar with the following `Stash` concepts:
+  - [BackupConfiguration](/docs/concepts/crds/backupconfiguration.md)
+  - [BackupSession](/docs/concepts/crds/backupsession.md)
+  - [Repository](/docs/concepts/crds/repository.md)
+
+Here, we are going to take backup from the `prod-1`, `prod-2` and `prod-3` namespaces and manage the backup from `backup` namespace using Auto Backup.
+
+Let's create the above-mentioned namespaces,
+
+```bash
+$ kubectl create ns prod-1
+namespace/prod-1 created
+
+$ kubectl create ns prod-2
+namespace/prod-2 created
+
+$ kubectl create ns prod-3
+namespace/prod-3 created
+
+$ kubectl create ns backup
+namespace/backup created
+
+```
+
+>**Note:** YAML files used in this tutorial can be found [here](https://github.com/stashed/docs/guides/managed-backup/dedicated-backup-namespace-auto-backup/examples).
+
+## Prepare Backup Blueprint
+
+In this section, we are going to prepare our Backup Blueprint object.
```

Comment:
Stash allows defining a backup template using `BackupBlueprint` CR. In this section, we are going to configure a `BackupBlueprint` for taking backup of MySQL databases.